scripts/v1alpha2/build.sh

```diff
@@ -29,4 +29,4 @@ echo "Building core image..."
 docker build -t ${PREFIX}/v1alpha2/katib-controller -f ${CMD_PREFIX}/katib-controller/v1alpha2/Dockerfile .
 docker build -t ${PREFIX}/v1alpha2/katib-manager -f ${CMD_PREFIX}/manager/v1alpha2/Dockerfile .
 docker build -t ${PREFIX}/v1alpha2/metrics-collector -f ${CMD_PREFIX}/metricscollector/v1alpha2/Dockerfile .
-
+docker build -t ${PREFIX}/v1alpha2/tfevent-metrics-collector -f ${CMD_PREFIX}/tfevent-metricscollector/v1alpha2/Dockerfile .
```

Comment:
please also enable image push in prow post/pre-submit
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha2/metrics-collector:${VERSION} --project=${PROJECT}
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha2/tfevent-metrics-collector:${VERSION} --project=${PROJECT}

Thanks!

Reply:
Done, thanks!